coforma/swift-tech-challenge
# Issue/Import and install SASS for front end styling

# Title
Issue/Import and install SASS for front end styling

# Summary

This imports SASS and connects it to USWDS. This will allow use to extend the USWDS styling with SASS and give us flexibility it how we style our components.

Fixes #18 

## Type of change


## Major Changes
This will require developers to re-run `yarn install` at the package root.

# How To Test
1. run `yarn dev`, then navigate to `localhost:3000` to verify the app is running successfully.
2. Edit the globals.scss file in the styles folder to see that you can change various styles!

# Checklist:
